include/Beman/Optional26/detail/stl_interfaces/iterator_interface.hpp

```diff
@@ -0,0 +1,986 @@
+// include/Beman/Optional26/detail/stl_interfaces/iterator_interface.hpp -*-C++-*-
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
This should be the Boost license identifier. Also, the top-level README should mention that some code is licensed under Boost and will be marked as such.

Reply:
Well, unless you could get @tzlaine to give you an "Apache-2.0 WITH LLVM-exception" license to his code. That would make it simpler.

Reply:
This is my official permission to use the Boost.STLInterfaces code under the Apache-2.0 WITH LLVM-exception license.

Reply:
That's great, @tzlaine! One more step closer to have a demo for Beman sync in Saint Louis.

Also, please check #21 (local demo with Godbolt and my fork of Beman.Optional26).